pkgs/tools/networking/circus/default.nix

```diff
@@ -1,7 +1,7 @@
-{ stdenv, buildPythonPackage, fetchPypi
```

Comment:
pass in python3 instead of the python packages

Reply:
trying to fix

Reply:
you mean add `python3Packages` to args and use referrence to `python3Packages.buildPythonPackage`?

Reply:
maybe done